docs/csi-addons-config.md

```diff
@@ -0,0 +1,26 @@
+# CSI-Addons Operator Configuration
+
+CSI-Addons Operator can consume configuration from a ConfigMap named `csi-addons-config`
+in the same namespace as the operator. This enables configuration of the operator to persist across
+upgrades. The ConfigMap can support the following configuration options:
```

Comment:
Should an empty ConfigMap be created by default? That gives users a hint where to place additional options, and they have an idea what to look for.

Reply:
Ok, if its easier to not have the configmap created by default, that is fine too.